### PR TITLE
State in handi popup when a station isn't without stairs.

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/lang/fr.json [hunk 1]
```diff
@@ -250,6 +250,7 @@
   "Naturerlebnispark": "Parc naturel périurbain",
   "Nein": "Non",
   "Nicht barrierefreier Bahnhof": "Gare non accessible",
+  "nicht treppenfrei": "Pas sans escalier",
```
Comment (oterral) [inline src/lang/fr.json]: @edellucien is that fench ? :)

Comment (edellucien) [reply src/lang/fr.json]: Litteral translation of 'nicht treppenfrei' or 'not without stairs' indeed.
But even in other language it seems weird.

File: src/popups/HandicapPopup/HandicapPopup.js [hunk 2]
```diff
@@ -32,6 +31,11 @@ function HandicapPopup({ feature }) {
 
   // build string for equipment
   const equipment = [];
+
+  equipment.push(
```
[the remark this answers is not in the document]
Comment (edellucien) [reply src/popups/HandicapPopup/HandicapPopup.js]:
```suggestion
  equipment.unshift(
```

File: src/lang/fr.json (hunk 1 above)
Comment (oterral) [reply src/lang/fr.json]: `Presence d'escaliers`  ou `avec escaliers` ou `Escaliers existant`

Comment (edellucien) [reply src/lang/fr.json]: tu manies la prose comme personne :smile:

**Pull request merged**